pkgs/development/python-modules/stringcase/default.nix

```diff
@@ -0,0 +1,29 @@
+{ buildPythonPackage, fetchPypi, stdenv
+}:
+
+buildPythonPackage rec {
+  pname = "stringcase";
+  version = "1.2.0";
+  name = "${pname}-${version}";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "023hv3gknblhf9lx5kmkcchzmbhkdhmsnknkv7lfy20rcs06k828";
+  };
+
+  checkInputs = [];
+
+  buildInputs = [];
+
+  propagatedBuildInputs = [];
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.